straxen/plugins/afterpulses/afterpulse_processing.py

```diff
@@ -153,6 +209,7 @@ def _find_ap(
 
     is_LED = False
     t_LED = None
+    t_LED_hit = None
```

Comment:
My understanding for those two times is:
- `t_LED_hit`: the weighted time of the first hit found, used for calculating the dynamic window -> only updated once for the first hit
- `t_LED`: the weighted time of all of the hits found in the window -> updated after reading every hit
If this is correct, could you consider adding some comments for that? Currently the names are a little bit confusing

Reply:
I renamed one variable and added comments explaining both.